python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/mpls.schema.yml

```diff
@@ -33,3 +33,249 @@ keys:
+            type: str
+            valid_values:
+              - implicit-null
+              - explicit-null
+          preemption_method:
+            type: dict
+            keys:
+              preemption:
+                type: str
+                required: true
+                valid_values:
+                  - hard
+                  - soft
+              timer:
+                description: Timer value in units of seconds.
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 65535
+          mtu_signaling:
+            description: Enable MTU signaling.
+            type: bool
+          graceful_restart:
+            description: RSVP graceful restart.
+            type: dict
+            keys:
+              role_helper:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                    required: true
+                  grace_period_recovery:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+                  grace_period_restart:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+              role_speaker:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                    required: true
+                  grace_period_recovery:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+                  grace_period_restart:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+          hello:
+            type: dict
+            keys:
+              interval:
+                description: Time between hello messages.
+                type: int
+                required: true
+                convert_types:
+                  - str
+                min: 1
+                max: 60
+              multiplier:
+                description: Number of missed hellos after which the neighbor is expired.
+                type: int
+                required: true
+                convert_types:
+                  - str
+                min: 1
+                max: 255
+          hitless_restart:
+            description: RSVP hitless restart.
+            type: dict
+            keys:
+              enabled:
+                type: bool
+              timer_recovery:
+                description: Time stale states will be preserved after restart.
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 320
+          p2mp_enabled:
```

Comment:
```
!
   p2mp
      disabled
```
p2mp_enabled will just render then `p2mp` command. Not sure that we should render the `disabled` command under p2mp or not.
cc: @ClausHolbechArista @gmuloc

Reply:
should be a dict - with a enabled key under the key.

if user gives input if False we should render disabled

Reply:
Fixed!